books/free-programming-books-es.md

```diff
@@ -281,6 +282,11 @@
 
 ### NoSQL
 
+#### DynamoDB
+
+* [Aprendizaje amazon-dynamodb](https://riptutorial.com/Download/amazon-dynamodb-es.pdf) - StackOverflow Contributors (PDF)
```

Comment:
```suggestion
* [Aprendizaje amazon-dynamodb](https://riptutorial.com/Download/amazon-dynamodb-es.pdf) - Compiled from StackOverflow documentation (PDF)
```

Reply:
trying to make this uniform across the repo. 
when you've made changes, please add `(changes made)` to the title of your PR